ui/src/testing/stubs/auth-domain-api-stub.ts

```diff
@@ -1,12 +1,8 @@
 import { AuthDomainApi } from 'generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class AuthDomainApiStub extends AuthDomainApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.